corehq/ex-submodules/couchexport/groupexports.py

```diff
@@ -54,7 +54,11 @@ def rebuild_export(config, schema, output_dir, last_access_cutoff=None, filter=N
             if saved.last_accessed is None:
                 saved.last_accessed = datetime.utcnow()
             saved.last_updated = datetime.utcnow()
-            saved.save()
+            try:
+                saved.save()
+            except ResourceConflict:
+                # task was executed concurrently, so let first to finish win and abort the rest
+                return
```

Comment:
(not blocking but using `try`/`else` might make the control flow here a little cleaner)